main.py

```
@@ -189,18 +188,18 @@ def tick_player(self, outside_temp_level, radiation_level):
         # updates hunger bar
         self.hungerbar = ""
         for i in range(math.ceil(self.hunger / 10)):
-            self.hungerbar += "\U0001f356"
+            self.hungerbar += "O"
         for i in range(10 - math.ceil(self.hunger / 10)):
-            self.hungerbar += "\U0000274c"
+            self.hungerbar += "X"
 
-        self.radiationbar = "\U00002622"
+        self.radiationbar = "Radiation:"
         for i in range(math.ceil(self.radiation / 100)):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)